dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -102,16 +98,25 @@ public static void cleanUp() {
   }
 
   @Test
-  public void testCss()
-      throws IOException, TemplateException, ParsingException, ArtifactDescriptorException {
+  public void testCss() {
     Path dashboardCss = outputDirectory.resolve("dashboard.css");
     Assert.assertTrue(Files.exists(dashboardCss));
     Assert.assertTrue(Files.isRegularFile(dashboardCss));
   }
 
+  private void assertDocument(String fileName, Consumer<Document> assertion)
+      throws IOException, ParsingException {
+    Path html = outputDirectory.resolve(fileName);
+    Assert.assertTrue(Files.isReadable(html));
+
+    try (InputStream source = Files.newInputStream(html)) {
+      Document document = builder.build(source);
```

Comment:
are we always asserting the same document? I think so, and if so we should simply store a Document field in setUp

Reply:
I still suspect Consumer and lambdas are overkill. This method can return the document and the specific tests can be run in the test methods.

Reply:
Updated. I was having wrong assumption that `Document` needs to be within a try clause.